flightgear.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-FG_ROOT=/app/share/flightgear FG_AIRCRAFT=$XDG_DATA_HOME/fgfs/Aircraft/ FG_SCENERY=$XDG_DATA_HOME/fgfs/Scenery fgfs --launcher "$@"
+FG_ROOT=/app/share/flightgear FG_AIRCRAFT=~/.fgfs/Aircraft/ FG_SCENERY=~/.fgfs/Scenery fgfs --launcher "$@"
```

Comment:
Ok, I've just tested the new build and it doesn't work as expected. I propose some changes: remove the FG_AIRCRAFT and FG_SCENERY environment variables, and set FG_ROOT to `~/.fgfs`. By default, aircrafts and scenery are stored in FG_ROOT/Aircraft and FG_ROOT/Scenery so it won't be of any problem. Then, replace the two `/app/share/flightgear` under `build-commands` of the `flightgear-data` module by this FG_ROOT, `~/.fgfs`. I now better understand how a `persist` directory in flatpak works. The specified directory will be relative to the home directory **in the sandbox**, but will appear on the host under `~/.var/app/$FLATPAK_ID/$PERSIST_PATH`. The persist directory will kept and restored in the sandbox each time it runs, making it available to the app.

Once those changes are made, it *should* work.